### PR TITLE
feat: implement token validation

### PR DESCRIPTION
**Pull request by anversa-pro**
## Describe your changes
Use event/:id/attendees endpoint to reuse the token verification with firebase SDK 
image

## Issue ticket number and link
https://trello.com/c/5bkWDhuA
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: providers/firebase-auth.application.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+dotenv.config()
+
+
+account.project_id = process.env.AUTH_PROJECT_ID
+account.private_key_id = process.env.AUTH_PRIVATE_KEY_ID
+account.private_key = JSON.parse(`"${process.env.AUTH_PRIVATE_KEY}"`)
+account.client_email = process.env.AUTH_CLIENT_EMAIL
+account.client_id = process.env.AUTH_CLIENT_ID
+account.client_x509_cert_url = process.env.AUTH_CLIENT_CERT_URL
```
Comment (jlora-mck) [inline providers/firebase-auth.application.js:18]: DRY DRY DRY (dont repeat yourself)

i made this comment in other PR, please dont repeat yourself you can use spread operator

`const account = {
...serviceAccount,
project_id = process.env.AUTH_PROJECT_ID,
.... //the rest
}`

File: providers/index.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
 'use strict'
 
-const setupFirebaseAdminSDKApp = require('./firebase-admin.application')
+// const setupFirebaseAdminSDKApp = require('./firebase-admin.application')
```
Comment (jlora-mck) [inline providers/index.js:3]: remove comments please

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please use current authentication file

File: providers/firebase-auth.application.js (hunk 1 above)
Comment (anversa-pro) [reply providers/firebase-auth.application.js]: Changes applied

File: providers/index.js (hunk 2 above)
Comment (anversa-pro) [reply providers/index.js]: Removed

**Comment by anversa-pro** [comment]
> current authentication file in usage

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Review by FelipeLosada1996 — APPROVED** [review]

**Review by jlora-mck — APPROVED** [review]

**Pull request merged**